mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/WEB-INF/src/content/Language.properties

```diff
@@ -7,7 +7,9 @@ contact-us=Contact us
 date-of-birth=Date of birth
 deleted-tasks-do-not-appear-in-history=Deleted tasks do not appear in history
 enter-task=Enter Task
-error=Error, form not submitted, you didn't fill out all the fields!
+error=Error, form not submitted, you didn't fill out all the fields
```

Comment:
@mateomustapic please change this into "Form not submitted, you did...." Two reasons: 1) error as a language key is too ambiguous/abstract 2) Background color of error message container and maybe error icon are standardized patterns in Liferay to display error message